src/soroban/friendbot.ts

```diff
@@ -0,0 +1,6 @@
+export namespace Friendbot {
```

Comment:
do you think it's worthwhile to hoist friendbot up to a top level folder since it represents a different physical server connection, following horizon, soroban-rpc?

Reply:
good idea!